pom.xml

```diff
@@ -246,8 +261,8 @@
       <groupId>org.opengeo.geoserver</groupId>
```

Comment:
@afabiani As I said in the GeoNode devel-list, I think we should be using this for the gs-geogig plugin:
```
<dependency>
    <groupId>org.geoserver.community</groupId>
    <artifactId>gs-geogig</artifactId>
    <version>${geoserver.version}</version>
</dependency>
```